.github/workflows/lint.yml

```diff
@@ -14,60 +14,32 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  lint-pycodestyle:
-    name: Code style check with pycodestyle
+  lint:
+    name: Lint
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
       uses: actions/checkout@v4
+
     - name: Merge CI fixes from sagemath/sage
       run: |
         .ci/merge-fixes.sh
       env:
         GH_TOKEN: ${{ github.token }}
+
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
         python-version: 3.9
-    - name: Install pycodestyle
-      run: pip install tox pycodestyle
-    - name: Lint using pycodestyle
+
+    - name: Install dependencies
+      run: pip install tox pycodestyle relint
```

Comment:
you don't need to manually install pycodestyle, relint.

tox installs them in its virtual environments automaticallly.